### ISSUE TITLE
Get the main file from the package.json

### ISSUE THREAD

**Issue by dannyfritz** [wscherphof/lua-loader#4]
I did it [here](https://github.com/dannyfritz/commonjs-lua/blob/01ed522/commonjs.lua#L6-L8) for reference.

**Comment by wscherphof** [comment]
Didn't you actually do the whole of what lua-loader did?

**Comment by dannyfritz** [comment]
Yes, i did end up going ahead with writing it after i made this issue. In the end I'm thinking i should get more comfy with luarocks --tree
